services/movie_session.py

```diff
@@ -0,0 +1,52 @@
+from datetime import datetime, timedelta
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> None:
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+
+
+def get_movies_sessions(
+        session_date: str = None
+) -> MovieSession:
+    if not session_date:
+        return MovieSession.objects.all()
+    else:
+        start_time = datetime.strptime(session_date, "%Y-%m-%d")
+        end_time = start_time + timedelta(days=1)
+        return MovieSession.objects.filter(
+            show_time__range=[start_time, end_time]
+        )
+
+
+def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(
+        session_id: int,
+        show_time: datetime = None,
+        movie_id: int = None,
+        cinema_hall_id: int = None
```

Comment:
use typehint like: Optional[YOUR_VALUE_TYPE] = None
for example: list that contains string elements and can be None has typehint Optional[list[str]]